importer/budgeteer-aproda-importer/build.gradle

```diff
@@ -1,5 +1,13 @@
+plugins {
+    id("de.adesso.budgeteer.java-conventions")
+}
+
 dependencies {
     implementation project(':budgeteer-importer-api')
     implementation group: 'org.apache.poi', name: 'poi-ooxml', version: poi_version
     runtimeOnly group: 'commons-codec', name: 'commons-codec', version: commons_codec_version
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junit5_version
```

Comment:
wäre es nicht sinnvoll, die Dependencies von Z.9 bis Z.12 auch in ein convention plugin aufzunehmen?